src/app/components/challenge/challengeleaderboard/challengeleaderboard.component.ts

```diff
@@ -98,6 +113,15 @@ export class ChallengeleaderboardComponent implements OnInit, AfterViewInit {
    */
   entryHighlighted: any = null;
 
+  /**
+   * Challenge phase visibility
+   */
+  challengePhaseVisibility = {
+    owner_and_host: 1,
+    host: 2,
+    public: 3,
```

Comment:
I see only `public` property used below in `filterPhases( )`.

Reply:
@lunayach I think it is better to at least list all the available options as they are same on the backend side